source/server/getting-started/configuration/index.rst

```diff
@@ -42,25 +42,69 @@ Dimension configuration files are used to affect a certain dimension or group of
 override the global config files. World configuration files are used to modify individual worlds only. World configs
 override dimension and global configs.
 
-It is possible to modify these configs through the in-game command ``/sponge conf``.
-
-The sponge conf command syntax looks like this:
-
-``/sponge conf <flag> <name> <key> <value>``
-
-The *flag* is the group you are targeting, such as global, dimension, or world.
+There are *flags* for specifying the target that you would wish to change. These flags are global, dimension, and world.
```

Comment:
Flags are explicitly related to the command. Moving the command down and leaving the flags here does not make any sense.